be/src/storage/async_delta_writer.cpp

```diff
@@ -83,11 +100,47 @@ void AsyncDeltaWriter::write(const AsyncDeltaWriterRequest& req, AsyncDeltaWrite
     task.indexes_size = req.indexes_size;
     task.write_cb = cb;
     task.commit_after_write = req.commit_after_write;
+    task.sync_write_task = false;
+    task.sync_write_finish_flag = nullptr;
     int r = bthread::execution_queue_execute(_queue_id, task);
     if (r != 0) {
         LOG(WARNING) << "Fail to execution_queue_execute: " << r;
         task.write_cb->run(Status::InternalError("fail to call execution_queue_execute"), nullptr);
     }
+    StarRocksMetrics::instance()->memtable_execution_queue_count.increment(1);
+}
+
+void AsyncDeltaWriter::sync_write(const AsyncDeltaWriterRequest& req, AsyncDeltaWriterCallback* cb) {
```

Comment:
I think it's better to use a request/response to limit bandwidth to send data.
It's better to consider the disk I/O performance to throttle.